src/App.js

```diff
@@ -1,13 +1,65 @@
 import React from 'react';
-
 import './App.scss';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodsList } from './GoodsList';
+
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  render() {
+    const { goods } = this.state;
+
+    return (
+      <>
+        <h1>Dynamic list of Goods</h1>
+
+        <GoodsList goods={goods} />
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+        <button
+          type="button"
+          onClick={() => {
+            getAll()
+              .then((res) => {
+                this.setState(state => ({
+                  goods: res,
+                }));
+              });
+          }}
+        >
+          Load All goods
+        </button>
+        <button
+          type="button"
+          onClick={() => {
+            get5First()
+              .then((res) => {
+                this.setState(state => ({
+                  goods: res,
+                }));
+              });
```

Comment:
You can make 1 function outside return statement, that will accept a callback as a parameter and use it for 3 click handlers